content/docs/querying/functions.md

```diff
@@ -80,6 +86,55 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the φ-quantile (0 ≤ φ
+≤ 1) from the buckets `b` of a histogram. The samples in `b` are the counts of
+observations in each bucket. Each value must have a label `le` where the label
+value denotes the inclusive upper bound of the bucket. (Samples without such a
+label are ignored.) The [histogram metric
+type](/docs/concepts/metric_types/#histogram) automatically
+provides time series with the `_bucket` suffix and the appropriate labels.
+
+Use the `rate()` function to specify the time window for the quantile
+calculation.
+
+Example: A histogram metric is called `http_request_duration_seconds`. To
+calculate the 90th percentile of request durations over the last 10m, use the
+following expression:
+
+```
+histogram_quantile(0.9, rate(http_request_duration_seconds_bucket[10m]))
+```
+
+The quantile is calculated for each label combination in
+`http_request_duration_seconds`. To aggregate, use the `sum()` aggregator
+outside of the `rate()` function. Since the `le` label is required by
```

Comment:
s/outside/around/ - an example maybe? Probably belongs in the rules best practices.